infrastructure/restapi/build.gradle

```diff
@@ -22,6 +22,9 @@ dependencies {
   testImplementation testFixtures(project(':infrastructure:json'))
 
   testFixturesImplementation testFixtures(project(':infrastructure:json'))
+  testFixturesImplementation project(':infrastructure:http')
+  testFixturesImplementation 'org.apache.logging.log4j:log4j-core'
```

Comment:
Why do we need log4j-core in test fixtures?  That feels wrong.  Maybe just log4j-api could work?

Reply:
i can try to tweak it, this was what i added when it finally worked... could potentially not log